### DISCUSSION TITLE
CUYANIMELIST || SESI 10 handle add to collection

### DISCUSSION THREAD

**Discussion by adiatmadl1001** [deaafrizal/forum#48] [Troubleshooting Room]
mohon bantuannya bang sebelum ditambahin authUserSession() di component anime/[id] masih aman aman aja trus pas udah di tambahin buat ngirim param id sama user?.email malah dapet error kek gini kenapa ya ??? udah aku coba di incognito window tetep aja, meskipun udah sign in juga tetep dapet error

![Screenshot 2024-01-02 233723](https://github.com/deaaprizal/forum/assets/73701124/ac80afb2-892a-4da6-81cb-4f22bcbde774)
![image](https://github.com/deaaprizal/forum/assets/73701124/8a836515-b129-4bea-bd39-5a5b5b9934b1)

**Comment by deaafrizal** [comment]
udah coba compare code ke repository gue bro?

**Reply by adiatmadl1001** [reply]
maaf bang🙏 ternyata ada salah di page.jsx nya, di punya gua client component yang harusnya server component jadi gua pisahin client componentnya udah solve hehe...😁
